src/inverseRobot.js

```diff
@@ -24,7 +24,24 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const obj = robot;
+  const newRobot = {};
+
+  for (const oldKey in obj) {
+    const newKey = obj[oldKey];
+
+    newRobot[newKey] = oldKey;
+  }
+
+  const size1 = Object.keys(robot).length;
+
+  const size2 = Object.keys(newRobot).length;
+
+  if (size1 !== size2) {
+    return null;
+  }
```

Comment:
All these checks are after the copy, you can improve your code by checking if the key is repeated right in the loop above, a little hint: `key in` or `hasOwnProperty()` would do the trick)